backend/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -59,6 +54,11 @@
       "name": "simple-report.sendgrid",
       "type": "gov.cdc.usds.simplereport.config.SendGridProperties",
       "description": "Properties for configuring SendGrid"
+    },
+    {
+      "name": "simple-report.authorization.environment-name",
+      "type": "java.lang.String",
+      "description": "Environment name used to derive Okta properties (role claim, admin group name, etc)."
```

Comment:
```suggestion
```
The `simple-report.authorization` dictionary is already called out as being bound to AuthorizationConfig up on line 14—no need to add new fields that are part of the definition of that object.